### PR TITLE
Allow SupportedProperty to overwrite a vocabulary's rdfs:label and rdfs:comment

### LINKED ISSUE #1: Allow SupportedProperty to overwrite a vocabulary's rdfs:label and rdfs:comment
**_This [was raised](http://lists.w3.org/Archives/Public/public-hydra/2013Jul/0004.html) by @matleh:**_

Another thought: If I want to use that information for form creation, I would like to have control over the label (and maybe even comment), but the label of the property might come from an extern ontology. Would it be feasible, to have a hydra:label that a form creation library could give precedence over rdfs:label?

### PR DESCRIPTION
**Pull request by alien-mcl**
I've added proper paragraphs to the specification that should formalize the _hydra:title_ and _hydra:description_ precedence over _rdfs:label_ and _rdfs:comment_ predicates, which should resolve #1

### CODE REVIEW

File: spec/latest/core/index.html [hunk 1]
```diff
@@ -560,6 +560,17 @@ <h3>Documenting a Web API</h3>
       <i>ApiDocumentation</i>.</p>
 
     <p class="issue">Describe the various properties of an operation.</p>
+
+    <p>Hydra allows also to document both <i>ApiDocumentation</i> and
+      hypermedia controls with human-readable descriptions by applying
+      <i>hydra:title</i> and <i>hydra:description</i> (as shown in the
```
Comment (tpluscode) [inline spec/latest/core/index.html]: I think we decided a while back to exclude the `hydra:` prefix in examples

```suggestion
      <i>title</i> and <i>description</i> (as shown in the
```

File: spec/latest/core/index.html [hunk 2]
```diff
@@ -560,6 +560,17 @@ <h3>Documenting a Web API</h3>
...
+      previous examples above). The former states a name of such a
+      decorated element that could be displayed to a human, the latter
+      provides it's description to be presented to that human i.e. as a hint.</p>
+
+    <p>Mentioned <i>hydra:title</i> and <i>hydra:description</i> SHOULD
```
Comment (tpluscode) [inline spec/latest/core/index.html]: Prefix, ditto

```suggestion
    <p>Mentioned <i>title</i> and <i>description</i> SHOULD
```

File: spec/latest/core/index.html [hunk 3]
```diff
@@ -560,6 +560,17 @@ <h3>Documenting a Web API</h3>
       <i>ApiDocumentation</i>.</p>
 
     <p class="issue">Describe the various properties of an operation.</p>
+
+    <p>Hydra allows also to document both <i>ApiDocumentation</i> and
```
Comment (tpluscode) [inline spec/latest/core/index.html]:
```suggestion
    <p>Hydra also allows documenting both <i>ApiDocumentation</i> and
```

https://ell.stackexchange.com/a/11197

File: spec/latest/core/index.html [hunk 4]
```diff
@@ -560,6 +560,17 @@ <h3>Documenting a Web API</h3>
...
+      hypermedia controls with human-readable descriptions by applying
+      <i>hydra:title</i> and <i>hydra:description</i> (as shown in the
+      previous examples above). The former states a name of such a
+      decorated element that could be displayed to a human, the latter
+      provides it's description to be presented to that human i.e. as a hint.</p>
+
+    <p>Mentioned <i>hydra:title</i> and <i>hydra:description</i> SHOULD
+      override standard <i>rdfs:label</i> and <i>rdfs:comment</i> and presented
```
Comment (tpluscode) [inline spec/latest/core/index.html]: It's not really overriding anything. 

```suggestion
      take precedence over standard <i>rdfs:label</i> and <i>rdfs:comment</i> and presented
```

I also question the word `standard` but it's not really incorrect.

File: spec/latest/core/index.html [hunk 5]
```diff
@@ -560,6 +560,17 @@ <h3>Documenting a Web API</h3>
       <i>ApiDocumentation</i>.</p>
 
     <p class="issue">Describe the various properties of an operation.</p>
+
+    <p>Hydra allows also to document both <i>ApiDocumentation</i> and
+      hypermedia controls with human-readable descriptions by applying
+      <i>hydra:title</i> and <i>hydra:description</i> (as shown in the
+      previous examples above). The former states a name of such a
```
Comment (tpluscode) [inline spec/latest/core/index.html]: Tautology

```suggestion
     examples above). The former states a name of such a
```

File: spec/latest/core/index.html [hunk 6]
```diff
@@ -560,6 +560,17 @@ <h3>Documenting a Web API</h3>
...
+      decorated element that could be displayed to a human, the latter
+      provides it's description to be presented to that human i.e. as a hint.</p>
```
Comment (tpluscode) [inline spec/latest/core/index.html]: "Displayed to a human", "presented to a human". 

It's a bit repetitive. Let's make this paragraph more concise

File: spec/latest/core/index.html (hunk 4 above)
Comment (alien-mcl) [reply spec/latest/core/index.html]: Maybe `common` would fit better?

Comment (tpluscode) [reply spec/latest/core/index.html]: No strong opinion here. We could keep it as is

File: spec/latest/core/index.html [hunk 7]
```diff
@@ -560,6 +560,17 @@ <h3>Documenting a Web API</h3>
       <i>ApiDocumentation</i>.</p>
 
     <p class="issue">Describe the various properties of an operation.</p>
+
+    <p>Hydra also allows documenting both <i>ApiDocumentation</i> and
+      hypermedia controls with human-readable descriptions by applying
+      <i>title</i> and <i>description</i> (as shown in the examples above).
+      The former states a name of such a decorated element that could
+      be displayed, the latter provides it's description
```
Comment (tpluscode) [inline spec/latest/core/index.html]:
```suggestion
      be displayed as a label, the latter provides its description
```

File: spec/latest/core/index.html (hunk 4 above)
Comment (alien-mcl) [reply spec/latest/core/index.html]: Than let's leave it as it is now.

File: spec/latest/core/index.html [hunk 8]
```diff
@@ -560,6 +560,17 @@ <h3>Documenting a Web API</h3>
...
     <p class="issue">Describe the various properties of an operation.</p>
+
+    <p>Hydra also allows documenting both <i>ApiDocumentation</i> and
+      hypermedia controls with human-readable descriptions by applying
+      <i>title</i> and <i>description</i> (as shown in the examples above).
+      The former states a name of such a decorated element that could
+      be displayed as a label, the latter provides its description
+      to be presented i.e. as a hint.</p>
+
+    <p>Mentioned <i>title</i> and <i>description</i> SHOULD take precedence
```
Comment (tpluscode) [inline spec/latest/core/index.html]:
```suggestion
    <p>Aforementioned <i>title</i> and <i>description</i> SHOULD take precedence
```

File: spec/latest/core/index.html [hunk 9]
```diff
@@ -560,6 +560,17 @@ <h3>Documenting a Web API</h3>
       <i>ApiDocumentation</i>.</p>
 
     <p class="issue">Describe the various properties of an operation.</p>
+
+    <p>Hydra also allows documenting both <i>ApiDocumentation</i> and
```
Comment (tpluscode) [inline spec/latest/core/index.html]:
```suggestion
    <p>Hydra also allows enriching both <i>ApiDocumentation</i> and
```

avoids repetition

File: spec/latest/core/index.html [hunk 10]
```diff
@@ -560,6 +560,17 @@ <h3>Documenting a Web API</h3>
...
+      hypermedia controls with human-readable descriptions by applying
+      <i>title</i> and <i>description</i> (as shown in the examples above).
+      The former states a name of such a decorated element that could
+      be displayed as a label, the latter provides its description
```
Comment (tpluscode) [inline spec/latest/core/index.html]:
```suggestion
      be displayed as a label. The latter provides its description
```

File: spec/latest/core/index.html [hunk 11]
```diff
@@ -560,6 +560,17 @@ <h3>Documenting a Web API</h3>
...
+      to be presented i.e. as a hint.</p>
+
+    <p>Mentioned <i>title</i> and <i>description</i> SHOULD take precedence
+      over standard <i>rdfs:label</i> and <i>rdfs:comment</i> and presented
+      to human if provided.</p>
```
Comment (tpluscode) [inline spec/latest/core/index.html]: > and presented to human if provided.

I think this is redundant

**Pull request merged**